src/tests/unit/tests/background/target-page-controller.test.ts

```diff
@@ -512,7 +592,8 @@ describe('TargetPageController', () => {
 
             describe('onConnect', () => {
                 it('should not have any observable effect', () => {
-                    setupDatabaseInstance(null, Times.never());
+                    idbInstanceMock.reset();
```

Comment:
Do we need to reset the mock here? it looks like it's already handled in the beforeEach one level up